src/libImaging/Pack.c

```diff
@@ -258,16 +258,6 @@ void
 ImagingPackRGB(UINT8 *out, const UINT8 *in, int pixels) {
     int i = 0;
     /* RGB triplets */
-#ifdef __sparc
```

Comment:
Could you say a few words about why [this](https://www.ibm.com/docs/en/clearcase/10.0.1?topic=example-defining-architecture-specific-cpp-macros) is removed?